Resources/SQLs/Updates/3_RenameUserStatsTable.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE user_stats

```

Comment:
Please append these changes to `Original Database.sql`

Reply:
Done #83